tests/README.md

```diff
@@ -116,7 +116,7 @@ at [mypy](https://github.com/python/mypy/issues).
 To run stubtest against third party stubs, it's easiest to use stubtest
```

Comment:
Want to move this paragraph into the stubtest_third_party section where you made the other change? (I think this text was written before we tested third party packages with stubtest, which is how this text ended up in the stdlib section)

Reply:
That sounds reasonable, I'll see about doing that pending figuring out the solution with the other PR.

Reply:
Resolved in 6d2bd8e